timesketch/frontend-ng/src/views/Sketch.vue

```diff
@@ -14,341 +14,345 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 <template>
-  <div v-if="sketch" style="height: 30%">
+  <div>
     <!-- Progress indicator when loading sketch data -->
     <v-progress-linear v-if="loadingSketch" indeterminate color="primary"></v-progress-linear>
 
-    <!-- Empty state -->
-    <v-container v-if="!hasTimelines && !loadingSketch" fill-height fluid>
-      <v-row align="center" justify="center">
-        <v-sheet class="pa-4">
-          <center>
-            <div style="font-size: 2em" class="mb-3">It's empty around here</div>
-            <ts-upload-timeline-form btn-size="normal" btn-type="outlined"></ts-upload-timeline-form>
-          </center>
-        </v-sheet>
-      </v-row>
-    </v-container>
+    <div v-if="sketch.id" style="height: 30%">
+      <!-- Empty state -->
+      <v-container v-if="!hasTimelines && !loadingSketch" fill-height fluid>
+        <v-row align="center" justify="center">
+          <v-sheet class="pa-4">
+            <center>
+              <div style="font-size: 2em" class="mb-3">It's empty around here</div>
+              <ts-upload-timeline-form btn-size="normal" btn-type="outlined"></ts-upload-timeline-form>
```

Comment:
The `It's empty around here` is currently hidden behind the top bar. If you make the windows smaller it will wrap and you nice that there is something behind it.

Reply:
Good catch, fixed now